drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/MetaDataColumnPageTest.java

```diff
@@ -70,27 +86,54 @@ public static void setupPreferences() {
 
         // Prevent runtime GWT.create() error at 'content = new SimplePanel()'
         GWTMockUtilities.disarm();
+
+        validationResult = new HashMap<>();
+        validationResult.put("metaData", true);
+        validationResult.put("*", false);
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        when(translationService.format(GuidedDecisionTableErraiConstants.MetaDataColumnPage_MetadataNameEmpty))
+                .thenReturn("empty");
+        when(translationService.format(GuidedDecisionTableErraiConstants.MetaDataColumnPage_ThatColumnNameIsAlreadyInUsePleasePickAnother))
+                .thenReturn("alreadyUsed");
+        when(translationService.format(GuidedDecisionTableErraiConstants.MetaDataColumnPage_IsNotValidIdentifier))
+                .thenReturn("isNotValid");
+
+        when(validationServiceCaller.call(any())).thenReturn(validationService);
```

Comment:
Using ```CallerMock``` (in ```uberfire-testing-utils```) is better/easier. You'd do ```validationServiceCaller = new CallerMock(validationService);``` and then your test won't need the ```RemoteCallback``` captor.